src/diffusers/pipelines/shap_e/camera.py

```diff
@@ -0,0 +1,133 @@
+from dataclasses import dataclass
```

Comment:
ok for me! That's extremely model specific, so I don't think we have to make the code super clean here